models/cow_protocol/ethereum/cow_protocol_ethereum_trades.sql

```diff
@@ -147,16 +147,33 @@ uid_to_app_id as (
     where i = j
 ),
 
+eth_flow_senders as (
```

Comment:
Since we only need sender and order_id, i think its more practical for us to simply construct the two column table directly here. The addition of sender column above is merely so that we aren't missing a crucial part of the ETH flow data in our view.